java/opendcs/src/main/java/decodes/tsdb/algo/TabRating.java

```diff
@@ -97,18 +97,16 @@ public void clearTable()
 		lookupTable.clear();
 	}
 
-	@Output public NamedVariable dep = new NamedVariable("dep", 0);
-	@Output String _outputNames[] = { "dep" };
-
-	@PropertySpec public boolean exceedLowerBound = false;
-	@PropertySpec public String tableDir = "$DECODES_INSTALL_DIR/tab-files";
-	@PropertySpec public String tableName = "";
-	@PropertySpec public boolean exceedUpperBound = false;
-	@PropertySpec public String tableNameSuffix = ".tab";
-	@PropertySpec String interp = "log"; // possibilities are log and linear
-	@PropertySpec public String nametype = "";
-	@PropertySpec public String _propertyNames[] = { "exceedLowerBound", "tableDir", "tableName", 
-			"exceedUpperBound", "tableNameSuffix", "interp", "nametype" };
+	@Output(type = Double.class)
+	public NamedVariable dep = new NamedVariable("dep", 0);
+
+	@PropertySpec(value = "false") public boolean exceedLowerBound = false;
```

Comment:
These should also be on line above. Or at least it would be nice if things just "read better".

Since you probably haven't looked at this in a few hours just scan it with your eyes again, and let your brain mull on what might make sense. I don't actually have a specific direction so just attempt some judgement if you think of something that seems better than spec on line above.